.circleci/config.yml

```diff
@@ -74,9 +74,8 @@ variables:
         if command -V docker >/dev/null; then
           mkdir -p /tmp/artifacts/images
           cd /tmp/artifacts/images
-          docker image ls --format '{{.Repository}}:{{.Tag}}' | \
-            (grep biocontainers || true) | \
-            xargs -n1 -P4 bash -c 'docker save $1 | gzip -c > "${1##*/}.tar.gz"' --
+          docker image ls --filter=reference='biocontainers/*'--format='{{.Repository}}:{{.Tag}}' | \
+            xargs -n1 -P4 bash -c 'test -n "${1+x}" && docker save "${1}" | gzip -c > "${1##*/}.tar.gz"' --
```

Comment:
We may not want to use the simple `xargs -r` here because `-r` is not POSIX.
(Although `docker` is likely only available on Linux... But just in case ;P. )